### PR TITLE
Convert ImageStoreManagerTest to Kotlin

### PR DESCRIPTION
**Pull request by megatunger**
## Summary:

Converts ImageStoreManagerTest.java to Kotlin, as per issue https://github.com/facebook/react-native/issues/37708

## Changelog:

[Internal] [Changed] - Convert ImageStoreManagerTest.java to Kotlin

## Test Plan:

Tests pass: ./gradlew :packages:react-native:ReactAndroid:test
Formatted with [KtFmt](https://facebook.github.io/ktfmt/)

### CODE REVIEW

**Comment by megatunger** [comment]
Hi @cortinico @mdvacca, Can you review this?

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import android.util.Base64
```
Comment (cortinico) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt:1]: As mentioned above:

```suggestion
package com.facebook.react.modules.camera

import android.util.Base64
```

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+class ImageStoreManagerTest {
+
+  private lateinit var reactApplicationContext: ReactApplicationContext
+  private lateinit var imageStoreManager: ImageStoreManager
+
+  @Before
+  fun setUp() {
+    reactApplicationContext = mock(ReactApplicationContext::class.java)
+    imageStoreManager =  ImageStoreManager(reactApplicationContext)
+  }
```
Comment (cortinico) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt:27]: +1 for doing this 👍

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+  private fun invokeConversion(inputStream: InputStream): String? {
+    return ImageStoreManager(reactApplicationContext).convertInputStreamToBase64OutputStream(inputStream)
+  }
```
Comment (cortinico) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt:31]:
```suggestion
  private fun invokeConversion(inputStream: InputStream): String? =
    imageStoreManager.convertInputStreamToBase64OutputStream(inputStream)
```

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+
+  private fun generateRandomByteString(length: Int): ByteArray? {
```
Comment (cortinico) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt:34]: Let's move all the `private` methods at the end after all the tests?

File: packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    val r = Random()
+    val sb = StringBuilder()
+    for (i in 0 until length) {
```
Comment (cortinico) [inline packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/camera/ImageStoreManagerTest.kt:37]:
```suggestion
    repeat(length) {
```

**Review by cortinico — CHANGES_REQUESTED** [review]
Thanks for sending this @megatunger 

I've left some comments that needs to be addressed before we can merge this. Thanks for doing this work.

**Comment by megatunger** [comment]
Hi @cortinico, thanks for your review. I updated them, can you take a look again?

**Review by cortinico — APPROVED** [review]

**Pull request closed without merging**